ksp_plugin/flight_plan.hpp

```diff
@@ -0,0 +1,85 @@
+﻿#pragma once
+
+#include <vector>
+#include <stack>
+
+#include "ksp_plugin/burn.hpp"
+#include "ksp_plugin/fork_handle.hpp"
+#include "ksp_plugin/frames.hpp"
+#include "ksp_plugin/manœuvre.hpp"
+#include "ksp_plugin/plugin.hpp"
+#include "physics/discrete_trajectory.hpp"
+
+namespace principia {
+
+using integrators::AdaptiveStepSizeIntegrator;
+
+namespace ksp_plugin {
+
+using NavigationManœuvre = Manœuvre<Barycentric, Navigation>;
+
+class FlightPlan {
+ public:
+  FlightPlan(
```

Comment:
Comment the parameters of the constructor.  Maybe not one by one, but at least by bunch (eg the stuff related to the integrator).